### ISSUE TITLE
Don't indent head & body tag

### ISSUE THREAD

**Issue by fmeilleur** [ngeor/html-fmt#10]
Not exactly an issue.
Is there an option to prevent the formatter from indenting the `<head>` & `<body>` tag ?

Thank you.

**Comment by ngeor** [comment]
Hi, no such option doesn't exist at the moment. I can implement it.

**Comment by fmeilleur** [comment]
If that's not too hard, it would be great !

**Comment by ngeor** [comment]
It should be available in the new version of the extension (0.8.0)

The snippet you need to add in `settings.json` is like this:

```
    "html-fmt-vscode.extraNonIndentingTags": [
        "html"
    ]
```
